frontend/public/components/dashboard/dashboard-card/card.scss

```diff
@@ -13,6 +13,10 @@
   justify-content: space-between;
 }
 
+.co-dashboard-card__see-all {
+  margin: -6.6px;
```

Comment:
Rather than using a negative margin with an arbitrary value, I recommend the following:
```css
line-height: 1;
padding: 0;
```

That aside, I think underlying issue is in the markup.  This element should actually be an `<a>` instead of a `<button>`.

But with that markup change, you'd need the following to correct the styling of the link:
```css
font-size: 14px;
font-weight: normal;
line-height: 1;
```

However, this really shouldn't be necessary.  PatternFly should support the inclusion of a link in the header.  So this is really an upstream bug.

Reply:
Since PatternFly doesn't correctly support the inclusion of a link in the card header, maybe the correct fix until the issue can be addressed upstream is to simply remove the link since this is just a demo?

Reply:
We have a `btn-link--no-btn-default-values` class that might help here. `button` is better for accessibility even if it's harder to style.

Reply:
@rhamilto Thank you for the suggestion, the style is now updated with `line-height` and `padding` instead of negative `margin`. 

It's true that the `see all` link is now used only by demo-plugin, but kubevirt and strorage will add their own subsystems soon.

Should I create a bug in PF ?

Reply:
@spadgett `btn-link--no-btn-default-values` doesn't help 

![see-all-btn](https://user-images.githubusercontent.com/2078045/60212812-6af26c00-9862-11e9-9bcf-d1ce4178e8ca.png)

Reply:
> Should I create a bug in PF ?

I think so.  It seems like this should be included in PF.